### PR TITLE
mdata-docs#1 Migrate away from Joyent

### PR DESCRIPTION
**Pull request by bahamat**

### CODE REVIEW

File: mdata/datadict.md [hunk 1]
```diff
@@ -169,36 +169,36 @@ the creation phase of zero-touch custom image management.
...
 
 [5]: https://en.wikipedia.org/wiki/Universally_unique_identifier
 
 [6]: protocol.html
 
-[7]: https://github.com/joyent/triton-cns
+[7]: https://github.com/tritondatacenter/triton-cns
 
-[8]: https://smartos.org/man/1M/mdata-list
+[8]: https://smartos.org/man/8/mdata-list
```
Comment (ncalvas) [inline mdata/datadict.md:204]: This link returned not-found.

**Review by ncalvas — CHANGES_REQUESTED** [review]
All looks good except that SmartOS manpage link.

File: mdata/datadict.md (hunk 1 above)
Comment (bahamat) [reply mdata/datadict.md:204]: Yeah, but it’s the new canonical place. I’ll be updating the target so that the link will work.

Comment (ncalvas) [reply mdata/datadict.md:204]: Ah ok. Then all is good!

**Review by ncalvas — APPROVED** [review]
Good for merging!

**Review by nwilkens — APPROVED** [review]

**Pull request merged**